locale/fof-follow-tags.yml

```diff
@@ -0,0 +1,41 @@
+fof-follow-tags:
+  forum:
+    settings:
+      notify_new_discussion_label: Iemand start een discussie in een tag die ik volg
+      notify_new_post_label: Iemand post in een tag die ik volg
+
+
+    index:
+      follow_tag_button: Volgen
+      unfollow_tag_button: Stop Volgen
```

Comment:
Wouldn't it better to translate it to "Stop met volgen" ?

Reply:
yes, I can change that.